Node/src/TeamsChatConnector.ts

```diff
@@ -48,18 +48,16 @@ export interface IInvokeEvent extends builder.IEvent {
   value: any;
 }
 
-export interface IntentMessage extends builder.IMessage {
-  intentText?: string;
-}
-
 export class TeamsChatConnector extends builder.ChatConnector {
 
   private allowedTenants: string[];
 
   private queryHandlers: { [id: string]: ComposeExtensionQueryHandlerType } = {};
 
+  private usingIntentText: boolean = false;
```

Comment:
Without IntentMessage.intentText this name doesn't make sense, since we don't have "intentText" anywhere. (what does it mean to "use intent text"?)